ice.c

```diff
@@ -3145,6 +3175,13 @@ void *janus_ice_send_thread(void *data) {
 			}
 			video_rtcp_last_sr = now;
 		}
+		/* Should we clean up old NACK buffers? */
+		if(max_nack_queue > 0 && (now-last_nack_cleanup >= (max_nack_queue/4))) {
```

Comment:
I think you need to multiply `max_nack_queue` by 1000 when comparing to `now` etc, like you do in `janus_cleanup_nack_buffer()`